README.md

```diff
@@ -81,8 +81,34 @@ scopelint check
 forge script script/Propose.s.sol --sig "run(address)" NEW_GOVERNOR_ADDRESS --rpc-url http://localhost:8545 --broadcast
  ```
 
+## Dependencies
+
+The Gitcoin Bravo governor inherits from `GovernorCountingFractional`, which is
+defined in the [flex-voting](https://github.com/ScopeLift/flexible-voting)
+project. This is done to enable:
+
+* partial voting (voting with less than full weight)
+* rolling voting (voting multiple times on the same proposal with partial weight)
+* split voting (splitting vote weight across against/for/abstain options)
+
+We use the `v1.0.0` tag of flex-voting because this was the version audited by
+Open Zeppelin.
+
+Additionally, this project depends on [Open Zeppelin's contracts
+library](https://github.com/OpenZeppelin/openzeppelin-contracts/). However,
+because of [an open issue in foundry](https://github.com/foundry-rs/foundry/issues/1855),
+we are currently sourcing the OZ library from within the flex-voting dependency.
+flex-voting [uses
+OZ](https://github.com/ScopeLift/flexible-voting/tree/4399694c1a70d9e236c4c072802bfbe8e4951bf0/lib)
+tagged at [v4.8.0](https://github.com/OpenZeppelin/openzeppelin-contracts/releases/tag/v4.8.0).
+
+At the time of writing (March 28 2023) [there are no
+changes](https://github.com/OpenZeppelin/openzeppelin-contracts/compare/49c0e43...d00acef) to any of the OZ
+contracts used by this repository between v4.8.0 and the [latest
+release, v4.8.2](https://github.com/OpenZeppelin/openzeppelin-contracts/releases/tag/v4.8.2).
+
 ## License
 
 The code in this repository is licensed under the [GNU Affero General Public License](LICENSE) unless otherwise indicated.
 
-Copyright (C) 2022 Gitcoin Core
+Copyright (C) 2023 Gitcoin Core
```

Comment:
Hmm, is this + the first two lines of the license correct? We're now working with Gitcoin DAO, this seems like it was from Gitcoin Core. For reference the first two lines of the license file are:

```
Copyright (C) 2022 Gitcoin Core
founders@gitcoin.co
```

Reply:
The latest Grants Stack repo has the same blurb at the top so I think it should be fine.

https://github.com/gitcoinco/grants-stack/blob/main/LICENSE